run_analysis.py

```diff
@@ -164,6 +164,15 @@ def analysis(config):  # {{{
             import moc_streamfunction
         moc_streamfunction(config)
 
+    if checkGenerate(config, analysisName='meridionalHeatTransport',
+                     mpasCore='ocean',
+                     analysisCategory='meridionalHeatTransport'):
```

Comment:
This is fine for now.  Eventually, we will have `tags` instead of `analysisCategory` and you can give a given analysis task multiple tags.  Then, it might make sense to give this task some tags that are common to other tasks.  Just something to keep in mind for later.